### DISCUSSION TITLE
get address transaction history

### DISCUSSION THREAD

**Discussion by MU12351235** [polkascan/py-substrate-interface#341] [Q&A] [answered]
Hi
Hope you feel good
I am working on a Polkadot related project
Now I want to get an address deposits history
Please help me how to do it?

**Comment by MU12351235** [comment]
Please help me..

**Comment by arjanz** [comment]
There is no index available on the Substrate node for transactions (extrinsics) per account. So you will need to use a third-party service that provides this indexed data, such as [Subsquid](https://www.subsquid.io/). 

I believe they have some [archives](https://app.subsquid.io/archives) for public use.

**Comment by arjanz** [comment, marked as answer]
Recently, an extension framework is added to the library, adding functionality that was not available or very inefficient to query directly through the Substrate RPC, such as transactions per account. Check out the [SubsquidExtension](https://polkascan.github.io/py-substrate-interface/extensions/subsquid-extension/) for example.
